src/Controls/samples/Controls.Sample.Droid/Maui.Controls.Sample.Droid-net6.csproj

```diff
@@ -15,4 +16,30 @@
     <PackageReference Include="Xamarin.AndroidX.Browser" />
   </ItemGroup>
 
+  <Target Name="_StaticWebAssetsComputeFilesToBuild" AfterTargets="GetCopyToOutputDirectoryItems" DependsOnTargets="ResolveStaticWebAssetsInputs">
+    <ItemGroup>
+      <_ExternalBuildStaticWebAsset Include="%(StaticWebAsset.FullPath)" Condition="'%(SourceType)' != ''">
+        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        <TargetPath>$([MSBuild]::MakeRelative('$(MSBuildProjectDirectory)','$([MSBuild]::NormalizePath('wwwroot\%(BasePath)\%(RelativePath)'))'))</TargetPath>
+      </_ExternalBuildStaticWebAsset>
+      <_SourceItemsToCopyToOutputDirectory Remove="@(_ExternalBuildStaticWebAsset)" />
+      <_SourceItemsToCopyToOutputDirectory Include="@(_ExternalBuildStaticWebAsset)" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="ZipWWWRootPath" AfterTargets="_FindJavaStubFiles" BeforeTargets="_CreateBaseApk" DependsOnTargets="_StaticWebAssetsComputeFilesToBuild">
+    <MakeDir Directories="$(ProjectDir)Assets" />
+    <ZipDirectory SourceDirectory="$(ProjectDir)$(OutputPath)wwwroot" DestinationFile="$(ProjectDir)Assets\wwwroot.zip" Overwrite="true" />
+  </Target>
+
+  <Target Name="_StaticWebAssetsUpdateAssetPaths" AfterTargets="_FindJavaStubFiles" BeforeTargets="_CreateBaseApk" DependsOnTargets="ZipWWWRootPath">
+    <ItemGroup>
+      <WWWRootZipStaticWebAsset Include="Assets\wwwroot.zip" />
+      <AndroidAsset Include="@(WWWRootZipStaticWebAsset)" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="DeleteAsset" AfterTargets="IncrementalClean" DependsOnTargets="ZipWWWRootPath">
+    <Delete Files="$(ProjectDir)Assets\wwwroot.zip" />
+  </Target>
```

Comment:
Is this something that will appear on the template (almost sure the answer is no, but have to ask 😄 )

Reply:
It won't 😄